src/System.Text.Json/tests/Serialization/DictionaryTests.KeyPolicy.cs

```diff
@@ -130,6 +130,46 @@ public static void NullNamePolicy()
             Assert.Equal(1, obj["onlyKey"]);
         }
 
+        [Fact]
+        public static void CustomNameSerialize_NullableValue()
```

Comment:
How about adding one where we don't have a `DictionaryKeyPolicy`.

Reply:
https://github.com/dotnet/corefx/blob/5e102fb5d30af4bc940aa519195c0e3dbd933648/src/System.Text.Json/tests/Serialization/NullableTests.cs